rcjaRegistration/events/admin.py

```diff
@@ -26,13 +27,42 @@
 class DivisionCategoryAdmin(AdminPermissions, admin.ModelAdmin):
     pass
 
+class ActiveFilter(admin.SimpleListFilter):
+    title = _('Active')
+
+    parameter_name = 'active'
+
+    def lookups(self, request, model_admin):
+        return (
+            ('active', _('Active')),
```

Comment:
When overriding the default filter need to make the default have a lookup value of None otherwise it won't show the selected filter correctly, see below.

![image](https://github.com/user-attachments/assets/66424625-61a8-49ef-bfa6-e1e6f9760691)